src/Controls/src/Core/Platform/Android/BottomNavigationViewUtils.cs

```diff
@@ -167,16 +167,10 @@ void clickCallback(object s, EventArgs e)
 
 					image.ImageTintList = ColorStateList.ValueOf(Colors.Black.MultiplyAlpha(0.6f).ToNative());
 
-					ImageSourceLoader shellImagePart = new ImageSourceLoader()
+					shellContent.icon.LoadImage(mauiContext, result =>
```

Comment:
All of the changes involving `ImageSourceLoader` is just me moving this behavior `ImageViewExtensions`